content_api/views.py

```diff
@@ -107,37 +95,19 @@ class GetPostsByAuthor(BasePostListRetrieval):
     lookup_url_kwarg = "id"
 
     def get_queryset(self):
-        _id = self.kwargs.get(self.lookup_url_kwarg)
-        posts = Post.objects.filter(author_id = _id)
-
-        # TODO This should go in get_object(self), but its not being called
```

Comment:
get_object is only for "retrieve" style calls. get, update, delete. Not list calls that return multiple objects.